data/ibilalkayy.json

```diff
@@ -38,6 +42,11 @@
       "url": "https://ibilalkayy.github.io/digital-resume/",
       "icon": "FaGlobe"
     },
+    {
+      "name": "Book Loop: A Startup",
+      "url": "https://bookloop.club",
```

Comment:
I think there is a problem with your website. PLease check.

Reply:
Yeah @priyankarpal. The DNS is broken but the domain is correct. It is going to be recovered soon.